types/atlassian-crowd-client/atlassian-crowd-client-tests.ts

```diff
@@ -0,0 +1,44 @@
+import CrowdClient from "atlassian-crowd-client";
```

Comment:
I translate the library usage into TS.
https://github.com/ghengeveld/atlassian-crowd-client#usage